pages/test_analytics/importing_json.md.erb

```diff
@@ -1,13 +1,9 @@
 # Importing JSON
 
-<div class="Docs__wip-note">
-<p class="Docs__wip-heading">Test Analytics is an invite-only beta feature. Join the <a href="https://buildkite.com/test-analytics">waitlist</a>.</p>
-</div>
+By default, Test Analytics ships with [RSpec collector](/docs/test-analytics/ruby-collectors#rspec-collector). However, you you can also upload test results by importing JSON or [JUnit XML](/docs/test-analytics/importing-junit-xml).
```

Comment:
Let's move this into an include? And use it in all four places so we're only updating it once? 

While we're there, I don't really like either "ships with" or "by default". How about something along the lines of 

> We currently have collectors Ruby (Rspec and Minitest) and Javascript (JEST). owever, you you can also upload test results by importing JSON or [JUnit XML](/docs/test-analytics/importing-junit-xml).

Reply:
CC @meiqimichelle

Reply:
> collectors Ruby (Rspec and Minitest) and Javascript (JEST)
We'll have more info (as in: things to link to) on Minitest and JEST collectors before this PR can be rolled out, right?

Reply:
That's the theory!

Reply:
And then swap it out in the other 3 places, right?

Reply:
@plaindocs I've already hacked together a snippet variant that can be further customized & linked before we merge.